### PR TITLE
Maps in relevant rooms and areas for ship guns

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Does not include ship guns, only the mapping so I can stop getting fucking merge conflicts every day

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Comment by Butterrobber202** [comment]
_heavy breathing_

File: code/modules/paperwork/folders.dm [hunk 1]
```diff
@@ -151,4 +151,84 @@
...
+/obj/item/folder/envelope/attackby(obj/item/W, mob/user)
+	if(sealed)
+		sealcheck(user)
+		return
+	else
+		..()
+
+/obj/item/folder/envelope/zta
+	name = "leviathan zero-point artillery instructions"
+	desc = "A small envelope. This one reads 'SCC - CONFIDENTIAL'."
```
Comment (SleepyGemmy) [inline code/modules/paperwork/folders.dm:196]:
```suggestion
	desc = "A small envelope with \"SCC CONFIDENTIAL\" in bold text on the front."
```

File: code/modules/paperwork/folders.dm [hunk 2]
```diff
@@ -151,4 +151,84 @@
...
+	acquisition/power management array.<br><field>\
+	<b>Instructions:</b> In order to fire the Leviathan, the capacitors must first be coupled by lowering the activation lever. This coupling is, however, extremely energy intensive. \
+	Long term activation thus requires a hefty supply of electricity in the APC. Firing the Leviathan itself takes energy directly from the superconducting SMES specially built \
+	for the gun. Its firing cost is around 20 megawatts per shot, not counting power taken from the APC in idle mode.<br> \
+	To activate the Leviathan itself, two command staff must first couple the warp fields to the firing array by unlocking the safeties on the keycard authentication device. \
+	Once that is done, the key must be retrieved from its case in the Captain's office. It must then be placed in the activation terminal in the bridge. \
+	The key must then be twisted and, finally, the button to fire the Leviathan will be uncovered and it may be pressed.<field><br>\
+	<b>Subject Description:</b> The Leviathan's one and only intended purpose is to annihilate any single target\
```
Comment (SleepyGemmy) [inline code/modules/paperwork/folders.dm:216]:
```suggestion
	<b>Subject Description:</b> The Leviathan's one and only intended purpose is to annihilate any single target \
```

File: code/modules/paperwork/folders.dm [hunk 3]
```diff
@@ -151,4 +151,84 @@
...
+	that proves to be an existential threat to the SCCV Horizon and all of its valuable designs and crew. The subject achieves this by pushing and weaponizing \
```
Comment (SleepyGemmy) [inline code/modules/paperwork/folders.dm:217]:
```suggestion
	that proves to be an existential threat to the SCCV Horizon and all of its valuable designs and crew. The weapon achieves this by pushing and weaponizing \
```

File: code/modules/paperwork/folders.dm [hunk 4]
```diff
@@ -151,4 +151,84 @@
...
+	already understood warp drive technology to a scale only currently matched by phoron fusion bombs.<br> \
+	In layman's terms, an extreme amount of energy is at first used by several high-level warp generators, which instead of bending space around the ship, \
+	create a 'warp funnel' around the outer half of the weapon. Every bit of energy remaining is then fired out through the 'warp tunnel', \
```
Comment (SleepyGemmy) [inline code/modules/paperwork/folders.dm:220]:
```suggestion
	creates a \"warp funnel\" around the outer half of the weapon. Every bit of energy remaining is then fired out through the warp funnel, \
```

File: code/modules/paperwork/folders.dm [hunk 5]
```diff
@@ -151,4 +151,84 @@
...
+	accelerating it into a powerful beam travelling at several times the speed of light. <br>No currently known hull and shield configuration is able to withstand the attack: \
+	the beam is expected to inflict too much structural damage for any one vessel to continue fighting. <br>\
+	The power draw of a single shot is greater than a single jump operation into bluespace, meaning each use will leave the Horizon vulnerable to any other threats that remain.<br>\
+	To date, the prototype has not been tested against stars, planets or other supermassive targets. While its use should be limited to catastrophic scenarios, \
```
Comment (SleepyGemmy) [inline code/modules/paperwork/folders.dm:224]:
```suggestion
	To date, the prototype has not been tested against stars, planets, or other supermassive targets. While its use should be limited to catastrophic scenarios, \
```

**Review by SleepyGemmy — COMMENTED** [review]
code has some minor comments.
sprites look fine.
mapping looks fine, as far as i can see.

File: code/modules/paperwork/folders.dm [hunk 6]
```diff
@@ -151,4 +151,84 @@
...
 	if(!length(contents))
-		qdel(src)
\ No newline at end of file
+		qdel(src)
+
+/obj/item/folder/envelope
+	name = "envelope"
+	desc = "A thick envelope. You can't see what's inside."
+	icon_state = "envelope_sealed"
+	var/sealed = 1
```
Comment (SleepyGemmy) [inline code/modules/paperwork/folders.dm:160]:
```suggestion
	var/sealed = TRUE
```

File: code/modules/paperwork/folders.dm [hunk 7]
```diff
@@ -151,4 +151,84 @@
...
+		icon_state = "envelope[contents.len > 0]"
+
+/obj/item/folder/envelope/examine(mob/user)
+	. = ..()
+	to_chat(user, "The seal is [sealed ? "intact" : "broken"].")
+
+/obj/item/folder/envelope/proc/sealcheck(user)
+	var/ripperoni = alert("Are you sure you want to break the seal on \the [src]?", "Confirmation","Yes", "No")
+	if(ripperoni == "Yes")
+		visible_message(SPAN_NOTICE("[user] breaks the seal on \the [src], and opens it."))
```
[the remark this answers is not in the document]
Comment (NonQueueingMatt) [reply code/modules/paperwork/folders.dm:175]: Not needed, user is a mob

**Comment by NonQueueingMatt** [comment]
Fuck this

**Pull request closed without merging**